app/src/main/java/com/sopt/now/MainActivity.kt

```diff
@@ -1,15 +1,26 @@
 package com.sopt.now
 
+import UserData
 import android.os.Bundle
 import androidx.activity.enableEdgeToEdge
 import androidx.appcompat.app.AppCompatActivity
 import androidx.core.view.ViewCompat
 import androidx.core.view.WindowInsetsCompat
+import com.sopt.now.databinding.ActivityMainBinding
 
 class MainActivity : AppCompatActivity() {
+    private lateinit var binding: ActivityMainBinding
+    private lateinit var user : UserData
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
-        setContentView(R.layout.activity_main)
+        binding = ActivityMainBinding.inflate(layoutInflater)
+        setContentView(binding.root)
+
+        user = intent.getSerializableExtra("login") as UserData
+
+        binding.userNickname.text = user.nickname
+        binding.userId.text = user.id
+        binding.userPassword.text = user.pwd
```

Comment:
```suggestion
        with(binding) {
            userNickname.text = user.nickname
            userId.text = user.id
            userPassword.text = user.pwd
        }
```

이런 식으로 with 함수로 묶어서 사용할 수 있어요 !

Reply:
with 함수로 묶는 방법 ! 저도 하나 배워갑니당